src/platform/packages/shared/kbn-unified-metrics-grid/src/components/metrics_experience_grid.tsx

```diff
@@ -121,18 +128,21 @@ export const MetricsExperienceGrid = ({
           disabled={dimensions.length === 0}
           indices={[indexPattern]}
           timeRange={timeRange}
+          clearValuesSelection={() => dispatch(setValueFilters([]))}
```

Comment:
Same here. Perhaps you could also use the `onValuesChange` and pass an empty array
```suggestion
     const onClearValues = useCallback(() => onValuesChange([]))
     
     <ValuesSelector
          onClear={onClearValues}
```

Reply:
Have you checked this suggestion? Wdyt?

Reply:
Thanks, done ✅